content/docs/guides/price-schedules.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: 'Same Product, Multiple Price Schedules'
+section: 'Guides'
+priority: 6
+hidden: false
+---
+
+import { Link } from 'gatsby'
+
+Price schedules are one of the critical factors related to <Link to="main-concepts/product-visibility">product visibility</Link> and there are several ways that you can build relationships between products and price schedules.  One of the scenarios you may wish to achieve is one product with multiple price schedules.  _Why_ would you possibly want one product that had multiple different prices based on buyer/user?  Lets take a look at the example scenario below, and walk through a solution to achieve the desired result.
+
+> As a seller company, I sell two different memberships to my b2b ecommerce site.  The first is **Startup** and the second is **Enterprise**.  The **Enterprise** membership is considerably more expensive than the **Startup** membership, therefore I want to provide a slightly lower product pricing to my **Enterprise** customers.  Specifically, Jane Doe from Best Buy has signed up for an **Enterprise** membership, and John Deer from TechWorld has signed up for a **Startup** membership.
+
```

Comment:
Maybe not an issue but might be safer to use businesses that don't exist

Reply:
I thought about this.  Changed.